src/api/spec/controllers/webui/staging/workflows_controller_spec.rb

```diff
@@ -46,6 +47,8 @@
       it { expect(response).to redirect_to(staging_workflow_path(project.staging)) }
       it { expect(subject.managers_group).to eq(managers_group) }
       it { expect(flash[:success]).not_to be_nil }
+      it { expect(reviewer.group).to eq(managers_group) }
+      it { expect(reviewer).not_to be_nil }
```

Comment:
Maybe it's just me I would find 
```ruby
it { expect(project.relationships.where(group: managers_group, role: Role.find_by_title('reviewer')).to exist }
```
easier to understand:thinking:

Though I would move this to the model tests. Since it's testing model behaviour.

Reply:
Changed! But `.to exist` can not be used with an `ActiveRecord::AssociationRelation`. I've checked if there is one element.

Reply:
@bgeuken I think I did something wrong, I've tried again and it worked. :clap: Updated!